Makefile

```
@@ -67,7 +70,10 @@ test-coverage:
 	sleep 10
 	docker compose -f dev/docker-compose-integration.yml cp ./dev/provision.py spark-iceberg:/opt/spark/provision.py
 	docker compose -f dev/docker-compose-integration.yml exec -T spark-iceberg ipython ./provision.py
-	poetry run coverage run --source=pyiceberg/ -m pytest tests/ ${PYTEST_ARGS}
+	poetry run coverage run --data-file=.coverage.integration --source=pyiceberg/ -m pytest tests/ -m integration ${PYTEST_ARGS}
```

nit: to match the other command, also added `-v` to match `test-integration`
```suggestion
	poetry run coverage run --source=pyiceberg/ --data-file=.coverage.integration -m pytest tests/ -v -m integration ${PYTEST_ARGS}
```

---

Done